recipes/c4core/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Fetch auxiliary GitHub projects only needed to build c4core (they are not
+# shipped with the library!) Note c4core relies on git submodules, but since we
+# have downloaded a tarball, we need to fetch these repos manually.
+git clone https://github.com/biojppm/cmake.git --depth 1
```

Comment:
Please add these as traceable sources in `meta.yaml`. That means using a specific git commit (`git_url` and other `git_*` options might help here). 

Also note that submodules usually mean vendoring something, so we are going to need an explanation about why that is the only option. If it's not, then packaging those utilities separately might be necessary.